[INFO -- 2018-10-12 17:10:41]
Starting flip_value script
version: 1.1
[INFO -- 2018-10-12 17:10:41]
Initialization strings:
[INFO -- 2018-10-12 17:10:41]
(1) file /tmp/lreg/lreg
[INFO -- 2018-10-12 17:10:41]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-10-12 17:10:41]
initSignal:0
[INFO -- 2018-10-12 17:10:41]
endSignal:10
[INFO -- 2018-10-12 17:10:41]
faultModel:1
[EXCEPTION -- 2018-10-12 17:10:43]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-10-12 17:10:43]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-10-12 17:10:43]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec1801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad38fb in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7ad9d3a in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7ad9d95 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7ad9fe8 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b02f26 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x0000555555557b3c in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd870, __n=18446744073709551612) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x00005555555578d5 in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744073709551612) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x00005555555573ac in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd870, __n=18446744073709551612) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x00005555555570cd in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd870, __n=18446744073709551612) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x00005555555567bd in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd870, __n=18446744073709551612, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x000055555555628f in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd870, __x=std::vector of length -4, capacity -4 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x0000555555555900 in train (iterations=30000, N=2, stopsWhenStable=true) at lreg.cpp:165
#14 0x0000555555555dba in main (argc=3, argv=0x7fffffffde68) at lreg.cpp:225

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping value of field: __val
Fliping array at pos: 1
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Double bit-flip
base address to flip value: 0x7fffffffd538
address max offset: 8
Memory content after  bitflip:0x000x880x000x000x000x000x000x00
frame name: __GI_abort
symbol name: sigs
symbol filename: abort.c
symbol line: 51
value: {__val = {32, 34816, 0 <repeats 14 times>}}
value address: 0x7fffffffd530
Type: The type is a structure.
Type sizeof: 128
Field name: __val
Field Type: The type is an array.
Field Type sizeof: 128

[INFO -- 2018-10-12 17:10:43]
Fault Injection Successful after 2.9572203159332275s
[INFO -- 2018-10-12 17:10:44]
event type: exit
[EXCEPTION -- 2018-10-12 17:10:44]
exit code: no exit code available
